runtime/src/main/java/com/dylibso/chicory/runtime/internal/CompilerRuntimeLib.java

```diff
@@ -13,9 +12,16 @@
 import com.dylibso.chicory.wasm.InvalidException;
 import com.dylibso.chicory.wasm.types.FunctionType;
 
-public final class AotMethods {
+/**
+ * This class contains the runtime library for the compiler. It MUST remain backwards compatible
+ * so that older generated code can run on newer versions of the library.
+ * <p>
+ * Don't change the method signatures, and be careful with any behavioral changes.  If in doubt test
+ * against older versions of the generated code.
+ */
+public final class CompilerRuntimeLib {
```

Comment:
As discussed in the other issue, we should not make this change. There doesn't seem to be any benefit to doing so, and it has complexity and compatibility downsides.

Reply:
Ok.  I'll back that part out.